frontend/src/App.tsx

```diff
@@ -19,6 +21,10 @@ function App() {
             <Route path="details-page/:title/:postId" element={<DetailsPage />} />
             <Route path="signin" element={<SignIn />} />
             <Route path="signup" element={<SignUp />} />
+            <Route path="admin">
+              <Route path="users" element={<AdminUsers />} />
```

Comment:
This is good, similarly do we have a hack to eliminate the wrapper in the `app.tsx`? Would be a good one if we could get it, else we'll use the wrapper AdminContainer